packages/jsapi-utils/src/ViewportDataUtils.ts

```diff
@@ -2,7 +2,8 @@ import type { Key } from 'react';
 import clamp from 'lodash.clamp';
 import type { Column, Row, Table, TreeTable } from '@deephaven/jsapi-types';
 import Log from '@deephaven/log';
-import { KeyedItem, WindowedListData } from '@deephaven/utils';
+import type { WindowedListData } from '@deephaven/react-hooks';
```

Comment:
jsapi-utils depending on react-hooks feels wrong as well. I think moving the dependencies here to jsapi-components instead (those are where React components that use the jsapi live).

Reply:
@mofojed can you elaborate on this one? jsapi-components depends on jsapi-utils. Which dependencies were you thinking of moving to jsapi-components?

Reply:
I was able to remove the react-hooks dependency by reconfiguring some types